lib/components/Hidden/Hidden.docs.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Hidden from './Hidden';
+import Text from '../Text/Text';
+
+export default {
+  component: Hidden,
+  examples: [
+    {
+      label: 'Hidden on mobile',
+      render: () => (
+        <Hidden component="span" hideFor="mobile">
+          <Text baseline={false}>Hidden on mobile</Text>
```

Comment:
The `baseline={false}` is not really required, it's best to try and keep these docs code blocks as lean as possible.